sendRawEthRandom.c

```diff
@@ -35,6 +41,16 @@ int main(int argc, char *argv[])
 	struct ether_header *eh = (struct ether_header *) sendbuf;
 	struct sockaddr_ll socket_address;
 
+	cpu_set_t set;
+	int cpu_id = atoi(argv[1]);
+	CPU_SET(cpu_id, &set);
+
+	if (sched_setaffinity(getpid(), sizeof(set), &set) == -1) {
+		printf("Failed to call sched_setaffinity() for CPU %d\n",
+			cpu_id);
```

Comment:
Since `sched_setaffinity()` sets `errno`, I think we can use `perror()` instead of `printf()`.